src/components/instagram-embed.vue

```diff
@@ -3,7 +3,7 @@
     class="instagram-media"
     data-instgrm-permalink="https://www.instagram.com/mistermonut/?utm_source=ig_embed&amp;utm_campaign=loading"
     data-instgrm-version="14"
-    style="width: 100%"
+    style="width: 100%; min-width: 0px !important"
```

Comment:
did this not work without the `!important`? 😭

Reply:
yeah idk why it didn't........ i didn't really want to use the !important, but i couldn't find where the min-width was being set, so i tried overriding it here. 😙🔫